src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -19,21 +19,28 @@ import DanishMitIDAddressJwtSnippet from '../../../snippets/jwt-payloads/danish-
 
 ### MitID for citizens (with address lookup)
 
-Address lookups is available as an add-on for MitID.
+[Address lookups](/verify/e-ids/danish-mitid/#collecting-user-addresses) is available as an add-on for MitID.
 
 <DanishMitIDAddressJwtSnippet />
 
 ## Test users
 
 You create personal MitID test users at [https://pp.mitid.dk/test-tool/frontend](https://pp.mitid.dk/test-tool/frontend):
 
-1. Just use the autofill button and then change the details to you liking
+1. Just use the autofill button and then change the details to you liking.
```

Comment:
Not related to this PR, but:
```suggestion
1. Just use the autofill button and then change the details to your liking.
```

Reply:
Fixed in [0d717cf](https://github.com/criipto/docs/pull/218/commits/0d717cfc85493cafe4cb6e2042f9e8fc7ff737e6).